client/src/components/CustomSearchBar.js

```diff
@@ -33,18 +63,28 @@ const CustomSearchBar = (props) => {
         placeholder="Search for a product or friend"
         className={classes.searchTextField}
         value={state.searchVal}
-        onChange={(event) => setState({...state, searchVal: event.target.value})}
+        onChange={(event) => {
```

Comment:
We should not call on onChange but instead onBlur function. https://www.w3schools.com/tags/ev_onblur.asp#:~:text=The%20onblur%20attribute%20fires%20the,opposite%20of%20the%20onfocus%20attribute. if it is just an input function or onSubmit if it is a button

Reply:
We have a search button to search for custom search that's why. It can be changed but I don't think this is a mvp change